app/server/schemas.py

```diff
@@ -392,15 +392,17 @@ class OrganisationSchema(SchemaBase):
     default_lat = fields.Float()
     default_lng = fields.Float()
 
+    valid_roles = fields.Raw()
+
     require_transfer_card = fields.Boolean(default=False)
     default_disbursement = fields.Function(lambda obj: int(obj.default_disbursement))
     country_code = fields.Function(lambda obj: str(obj.country_code))
 
     token               = fields.Nested('server.schemas.TokenSchema')
 
-    users               = fields.Nested('server.schemas.UserSchema', many=True)
-    transfer_accounts   = fields.Nested('server.schemas.TransferAccountSchema', many=True)
-    credit_transfers    = fields.Nested('server.schemas.CreditTransferSchema', many=True)
+    #users               = fields.Nested('server.schemas.UserSchema', many=True)
```

Comment:
Is there a reason users, transfer accounts, and credit transfers are returned by this API? Was making it run quite slow on a fully loaded db

Reply:
With these fields commented out, hitting an org endpoint is pretty much instant, but when any of them is uncommented the endpoints become unusably slow. Like on my local instance with demo data, when I have just the `transfer_accounts` line uncommented, it takes 23s to load and it returns a 29MB JSON payload containing every single transfer